libc-test/semver/linux-musl.txt

```diff
@@ -1,4 +1,14 @@
 # TODO: musl.
+# linux-gnu.txt is alphabetized, but this file is not
```

Comment:
Sorted in https://github.com/rust-lang/libc/pull/3954, could you rebase once that merges and sort these in?

Reply:
Done